docs/book/src/quick-start.md

```diff
@@ -131,6 +131,7 @@ type GuestbookStatus struct {
 	Standby []string `json:"standby"`
 }
 
+// +kubebuilder:object:root=true
 type Guestbook struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
So, we not to add all markers added by default. See: https://github.com/kubernetes-sigs/kubebuilder/blob/1eccd8fbe9f53ffa117b6367976230eb1fe13b27/testdata/project-v3/api/v1/admiral_types.go could you please update this one?

Reply:
I added the status marker if the scoping required too, ping me and i add that too